crates/solidity/outputs/cargo/slang_solidity/Cargo.toml

```diff
@@ -44,10 +44,10 @@ infra_utils = { workspace = true }               # __REMOVE_THIS_LINE_DURING_CAR
 solidity_language = { workspace = true }         # __REMOVE_THIS_LINE_DURING_CARGO_PUBLISH__
 
 [dependencies]
-anyhow = { workspace = true, optional = true }
 ariadne = { workspace = true, optional = true }
 clap = { workspace = true, optional = true }
 metaslang_cst = { workspace = true }
+metaslang_graph_builder = { workspace = true }
```

Comment:
If this is getting added to the public API deps, we need to make sure we also publish the `metaslang_graph_builder` crate in our CI. Otherwise, this would fail publishing new versions.

https://github.com/manastech/slang/blob/68d4f0dc1a3303de6b2adba24a37cd638a8036ab/crates/infra/cli/src/commands/publish/cargo/mod.rs#L14-L18

Reply:
Same as above, this is now optional and behind the `cli` feature flag. I also removed the dependency from the other crates (`testlang` and `node_addon`s) since it's not needed for now.

I guess we will still need to publish the crate. Should I add it to the infra script? Is there anything else involved beyond adding it [here](https://github.com/manastech/slang/blob/main/crates/infra/cli/src/commands/publish/cargo/mod.rs#L16)?

Also, should I add changeset on this PR?

Reply:
> Is there anything else involved beyond adding it here?

I think that is it.

> Also, should I add changeset on this PR?

Changesets are only for user-visible changes. As this is an infra-only change, no need to do it.
We would add a changeset once we decide to expose these APIs publicly.

Reply:
With the last commits, the only user-visible change introduced by this PR is the exposure of the `Language::ROOT_KIND` constant and its JS/TS counterpart `Language.rootKind()`.

Reply:
I added a changeset. Hopefully I'm understanding this correctly :)

Reply:
Perfect!

Reply:
Now we just need to add the new crate to `USER_FACING_CRATES`.